docs/infrastructure/README.md

```diff
@@ -0,0 +1,62 @@
+# Overview
+
+Student Robotics has a variety of infrastrcture in order to support its mission.
+This document aims to provide an outline of the general structure.
+
+Currently our infrastructure as a whole does not have a designed owner, instead
+individual pieces are looked after by volunteer Teams or just by volunteers who
+have historically maintained them.
+
+## Website
+
+Our website consists of a main public facing site at https://studentrobotics.org
+as well as a collection of other sites which appear at various paths off the
+root domain. This runbook is an example of such a sub-site, as are the
+competitor-facing documentation which appears at [`/docs`][slash-docs].
+
+These sub sites (and the main website itself) are hosted as GitHub Pages, with a
```

Comment:
Not all the sites are GitHub pages, ie the IDE.

If the idea is to emphasise that things like the docs and comp site might _look_ like they're 1 site, but are actually separate, then that's probably worth explaining separately.

Reply:
I'm intending to use the term "sub site" here to mean the things which are _websites_ (i.e: things with pages) rather than web apps or other services. I'd also argue that such things are not part of our _website_.
I've attempted to hint at this in the next sentence by also mentioning "other services". I'd also hope that it was clear (from the subsequent sections) that we have plenty of things which aren't hosted on GitHub Pages.

Reply:
ce75d2f3f7caf3cacb6d0df46e874ab8c7f668a6